src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,54 @@
  */
 
 public class Plane {
+    private int length;
+    private int height;
+    private int width;
+    private double maxSpeed;
+    private String color;
 
+    public Plane(int length, int height, int width, double maxSpeed, String color) {
+        this.length = length;
+        this.height = height;
+        this.width = width;
+        this.maxSpeed = maxSpeed;
+        this.color = color;
+    }
+
+    public class PlaneBuilder {
+        private int length;
+        private int height;
+        private int width;
+        private double maxSpeed;
+        private String color;
+
+        public PlaneBuilder setLength(int length) {
+            this.length = length;
+            return this;
+        }
+
+        public PlaneBuilder setHeight(int height) {
+            this.height = height;
+            return this;
+        }
+
+        public PlaneBuilder setWidth(int width) {
+            this.width = width;
+            return this;
+        }
+
+        public PlaneBuilder setMaxSpeed(double maxSpeed) {
+            this.maxSpeed = maxSpeed;
+            return this;
+        }
+
+        public PlaneBuilder setColor(String color) {
+            this.color = color;
+            return this;
+        }
+
+        public Plane build() {
+            return new Plane(length, height, width, maxSpeed, color);
```

Comment:
how will we create new object of Plane?
write please code in comment to this message

Reply:
Sorry, no way( at first it was static inner class, but a cant pass tests. Than i changed to inner class, and haven't checked. And after this collision was made, when test passed, but id doesn't work( Its my fault(